engine/api/workflow_hook.go

```diff
@@ -259,12 +259,7 @@ func (api *API) postWorkflowJobHookCallbackHandler() service.Handler {
 			return sdk.WrapError(err, "cannot load workflow run")
 		}
 
-		pv, err := project.LoadAllVariablesWithDecrytion(tx, wr.Workflow.ProjectID)
-		if err != nil {
-			return sdk.WrapError(err, "cannot load project variable")
-		}
-
-		secrets, err := workflow.LoadSecrets(tx, api.Cache, nil, wr, pv)
+		secrets, err := workflow.LoadSecrets(ctx, tx, wr, nil)
```

Comment:
before we needed variables with decryption but we don't need decryption anymore ?